ui/src/pages/OBProxy/Detail/Overview/DetailConfig.tsx

```diff
@@ -48,12 +50,35 @@ export default function DetailConfig({ style, ...props }: DetailConfigProps) {
             })}
             {image || '-'}
           </Col>
-          <Col span={24}>
+          <Col span={8}>
             {intl.formatMessage({
               id: 'src.pages.OBProxy.Detail.Overview.23ED4374',
               defaultMessage: '服务类型：',
             })}
-            {serviceType || '-'}
+            {service?.type || '-'}
+          </Col>
+          <Col span={8}>
+            {intl.formatMessage({
+              id: 'src.pages.OBProxy.Detail.Overview.102C210B',
+              defaultMessage: '服务地址：',
+            })}
+            {service?.clusterIP || '-'}
+          </Col>
+          <Col span={8}>
+            {service?.type === 'LoadBalancer' ? (
+              <span>externalIP：{service?.externalIP || '-'}</span>
+            ) : service?.type === 'NodePort' ? (
+              <span>
+                {intl.formatMessage({
+                  id: 'src.pages.OBProxy.Detail.Overview.357D1CE4',
+                  defaultMessage: '端口映射：',
+                })}
+
+                {service.ports
+                  ?.map((portItem) => `${portItem.port}:${portItem.targetPort}`)
```

Comment:
The pair should be `item.port:item.nodePort` instead of `:targetPort`

Reply:
Fixed.